SU2_CFD/include/drivers/CDiscAdjSinglezoneDriver.hpp

```diff
@@ -127,4 +127,10 @@ class CDiscAdjSinglezoneDriver : public CSinglezoneDriver {
    */
   void SecondaryRecording(void);
 
+  /*!
+   * \brief gets Convergence on physical time scale, (deactivated in adjoint case)
+   * \param none
+   */
+  virtual bool GetTimeConvergence() const;
```

Comment:
The virtual specifier you should use here is `override` i.e. `bool GetTimeConvergence() const override`, this immediately says to anyone reading the code that you are overriding functionality inherited from a parent class.
`virtual` should be used on the first class in the hierarchy that defines the method.

Reply:
Okay I'm going to fix that.